### DISCUSSION TITLE
How can I keep seeing my inserted link after downloading my diagram?

### DISCUSSION THREAD

**Discussion by abxxch** [jgraph/drawio#2565] [Q&A]
Is there a way I can click on the text where I have inserted a link after having downloaded my diagram as a photo?

![image](https://user-images.githubusercontent.com/98042688/150170867-8008d0c2-1b75-43f6-b1d3-33942415a4ce.png)

**Comment by davidjgraph** [comment]
No, it's an image.

**Reply by abxxch** [reply]
What is the purpose of having the possibility to add a link then? I really need that tool.
Is there another way to have the link there?

**Comment by davidjgraph** [comment]
Images don't support links, this is a fundemental property of images.

**Reply by abxxch** [reply]
Thank you, I was able to convert my diagram to pdf and I was able to see the link.
